.github/workflows/ci.yml

```diff
@@ -251,7 +251,8 @@ jobs:
           - ":trino-cassandra"
           - ":trino-hive,:trino-orc"
           - ":trino-hive,:trino-parquet -P test-parquet"
-          - ":trino-mongodb,:trino-kafka,:trino-elasticsearch,:trino-elasticsearch -P test-stats"
+          - ":trino-mongodb,:trino-kafka,:trino-elasticsearch"
+          - ":trino-elasticsearch -P test-stats"
```

Comment:
Do you mean that this profile wasn't being run on CI? Or is it just to split jobs to make them finish faster?

I would prefer to find and fix the cause instead.

Reply:
just added description.

Reply:
Thanks - that's such a footgun. Thanks for taking care of this.

Reply:
while at this, do you understand why the profile name is `test-stats`?
what `TestElasticsearchBackpressure` has to do with stats? (statistics?)

we could rename it to something else


and also, i don't understand why the `TestElasticsearchBackpressure` is isolated https://github.com/trinodb/trino/pull/7423/files#r658244083

cc @MiguelWeezardo

Reply:
good question, will wait for @MiguelWeezardo to chime in

Reply:
> while at this, do you understand why the profile name is test-stats?
> what TestElasticsearchBackpressure has to do with stats? (statistics?)

JMX stats reported by backpressure handling are used in `TestElasticsearchBackpressure` to verify that it was correctly identified and handled. 

> and also, i don't understand why the TestElasticsearchBackpressure is isolated https://github.com/trinodb/trino/pull/7423/files#r658244083

Statistics sometimes returned 0 as if they were not being updated. Apparently if more than one stats bean gets registered in the JVM, it can lead to stats not being updated properly. The test @kokosing mentioned as an example had a similar kind of isolation in place, presumably to avoid multiple stats bean registrations.

Reply:
FWIW, I had seen a [similar issue](https://github.com/trinodb/trino/blob/master/plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergMetadataFileOperations.java#L63). but it still seems to work with other tests in same `mvn test` - may be because it's the only test on iceberg module that registers this mbean.

Reply:
@findepi @MiguelWeezardo any preference around current profile name vs `test-mbeans`? 

Ideally, I think it might be better to configure a different random prefix for mbeans through `ConnectorObjectNameGeneratorModule` during TrinoServer creation for that test.